### PR TITLE
Adding missing repos and make infinispan auth as external configurations

### PR DESCRIPTION
**Pull request by ricardozanini**

### CODE REVIEW

File: 03-kogito-travel-agency/src/main/resources/META-INF/hotrod-client.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+infinispan.client.hotrod.auth_username=${infinispan_username}
```
Comment (mswiderski) [inline 03-kogito-travel-agency/src/main/resources/META-INF/hotrod-client.properties:1]: will this work if these place holder values are empty?

File: 03-kogito-travel-agency/pom.xml [hunk 2]
```diff
@@ -14,6 +16,47 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
   </properties>
+  <repositories>
+    <repository>
+      <id>sonatype-nexus-snapshots</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```
Comment (mswiderski) [inline 03-kogito-travel-agency/pom.xml:22]: why do we need these extra repositories?

File: 04-kogito-visas/pom.xml [hunk 3]
```diff
@@ -70,6 +70,12 @@
       <groupId>org.kie.kogito</groupId>
       <artifactId>kogito-events-reactive-messaging-addon</artifactId>
       <version>${kogito.version}</version>
+     <exclusions>
+        <exclusion>
+          <groupId>io.smallrye.reactive</groupId>
+          <artifactId>smallrye-reactive-messaging-provider-1.0</artifactId>
```
Comment (mswiderski) [inline 04-kogito-visas/pom.xml]: same here

File: 03-kogito-travel-agency/pom.xml (hunk 2 above)
Comment (ricardozanini) [reply 03-kogito-travel-agency/pom.xml:22]: This repo is where the Quarkus snapshots are. Without it, I couldn't build the projects. If you changed to released versions, we can safely remove those.

File: 04-kogito-visas/pom.xml (hunk 3 above)
Comment (ricardozanini) [reply 04-kogito-visas/pom.xml]: This is a fix proposed by @cristianonicolai

File: 03-kogito-travel-agency/src/main/resources/META-INF/hotrod-client.properties (hunk 1 above)
Comment (ricardozanini) [reply 03-kogito-travel-agency/src/main/resources/META-INF/hotrod-client.properties:1]: Yes! The client will assume default values (in this case `auth` will be disabled).

**Comment by mswiderski** [comment]
@ricardozanini would you mind rebase on top of master as we just upgraded to 0.23.1 of quarkus

**Comment by ricardozanini** [comment]
@mswiderski done thanks!

File: 04-kogito-visas/pom.xml (hunk 3 above)
Comment (mswiderski) [reply 04-kogito-visas/pom.xml]: this is no longer needed as we already upgraded to quarkus 0.23 that uses kogito 0.4

File: 04-kogito-travel-agency/pom.xml [hunk 4]
```diff
@@ -48,6 +48,12 @@
       <groupId>org.kie.kogito</groupId>
       <artifactId>kogito-events-reactive-messaging-addon</artifactId>
       <version>${kogito.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>io.smallrye.reactive</groupId>
+          <artifactId>smallrye-reactive-messaging-provider-1.0</artifactId>
```
Comment (mswiderski) [inline 04-kogito-travel-agency/pom.xml]: this is no longer needed as we already upgraded to quarkus 0.23 that uses kogito 0.4

**Comment by ricardozanini** [comment]
@mswiderski removed unneeded exclusions

**Review by mswiderski — APPROVED** [review]

**Pull request merged**